product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -2,83 +2,120 @@
 
 #### Physical replication
 
-Copies all changes from a database to one or more standby cluster members by copying an exact copy of database disk blocks. While fast, this method has downsides. For example, only one master node can run write transactions. Also, you can use this method only where all cluster members are on the same major version of the database software, in addition to several other more complex restrictions.
+By making an exact copy of database disk blocks as they are modified to one or more standby cluster members, physical replication provides an easily implemented method to replicate servers. But there are restrictions on how it can be used. For example, only one master node can run write transactions. Also, the method requires that all cluster members are on the same major version of the database software, in addition to several other more complex restrictions.
 
-#### Read scalability
+#### Postgres cluster
 
-Can be achieved by introducing one or more read replica nodes to a cluster and have the application direct writes to the primary node and reads to the replica nodes. As the read workload grows, you can increase the number of read replica nodes to maintain performance.
+Traditionally, in Postgresql, a number of databases running on a single server is referred to as a cluster (of databases). This kind of Postgres cluster is not highly available. To get high availability and redundancy, you need a [PGD Cluster](#pgd-cluster).
 
-#### Recovery point objective (RPO)
+#### Quorum
 
-The maximum targeted period in which data might be lost due to a disruption in delivery of an application. A very low or minimal RPO is a driver for very high availability.
+When a [Raft](#Raft) [consensus](#consensus) is needed by a PGD cluster, there needs to be  a minimum number of voting nodes participating in the vote. This number is called a quorum. For example, with a 5 node cluster, the quorum would be 3 nodes in the cluster voting. A consensus would be 5/2+1 nodes, 3 nodes voting the same way. If there were only 2 voting nodes, then a consensus would never be established.
 
-#### Recovery time objective (RTO)
+#### Raft
 
-The targeted length of time for restoring the disrupted application.  A very low or minimal RTO is a driver for very high availability.
+Replicated, Available, Fault Tolerance. A consensus algorithm which uses votes from a quorum of machines in a distributed cluster to establish a consensus. PGD uses RAFT within groups (top level or local) to establish which node is the write leader. 
 
-#### Single point of failure (SPOF)
+#### Read scalability
 
-The identification of a component in a deployed architecture that has no redundancy and therefore prevents you from achieving higher levels of availability.
```

Comment:
@djw-m should we say 'introduce one or more read replica nodes, also known as subscriber only nodes in PGD, to a cluster..." to link the 2 terms?

Reply:
Not sure the terms are quite interchangable as it could be possible to implement read replicas in other ways.